docs/pages/reference/mesheryctl-commands.md

```diff
@@ -17,10 +17,10 @@ Meshery CLI commands are categorized by function, which are:
 
 - `mesheryctl` - Global flags and CLI configuration
 - `mesheryctl system` - Meshery Lifecycle and Troubleshooting
-- `mesheryctl mesh` - Service Mesh Lifecycle & Configuration Management: provisioning and configuration best practices
-- `mesheryctl perf` - Service Mesh Performance Management: Workload and service mesh performance characterization
-- `mesheryctl pattern` - Service Mesh Pattern Configuration & Management: Service mesh patterns and Open Application Model integration
-- `mesheryctl app` - Service Mesh Application Management
+- `mesheryctl mesh` - Cloud Native Lifecycle & Configuration Management: provisioning and configuration best practices
+- `mesheryctl perf` - Cloud Native Performance Management: Workload and service mesh performance characterization
+- `mesheryctl pattern` - Cloud Native Pattern Configuration & Management: Service mesh patterns and Open Application Model integration
```

Comment:
Missed a reference here.

Reply:
Thank you for pointing out, resolved it!